docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
```

Comment:
Change Dockerfile to another specific name, because other people may add macOS or Windows dockerfile later.

Reply:
Update it later.